test/test-applications/integrations/TestApplication.Http.NetFramework/TestServer.cs

```diff
@@ -0,0 +1,135 @@
+// <copyright file="TestServer.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading;
+using TestApplication.Http.NetFramework.Helpers;
+
+namespace TestApplication.Http.NetFramework;
+
+public class TestServer : IDisposable
```

Comment:
As this class is copy paste, should changes from #1634 should be included here?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1628/commits/50384909e462141a7bbb6c27bdf6d2b70611a04f